lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -223,6 +224,9 @@ func (handler *ContainersHandlersImpl) RemoveContainerHandler(params containers.
 
 	err := h.Container.Remove(context.Background(), handler.handlerCtx.Session)
 	if err != nil {
+		if strings.Contains(err.Error(), "The attempted operation cannot be performed in the current state (Powered on)") {
```

Comment:
That message is localized, so it is best to check the error type instead.  In this case you could use the following helper:

```go
func isInvalidPowerStateError(err error) bool {
	if terr, ok := err.(task.Error); ok {
		_, invalid := terr.Fault().(*types.InvalidPowerState)
		return invalid
	}
	return false
}
```